__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require('../db/data/test-data/index');
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe('/api/topics', () => {
+  describe('GET', () => {
+    it('responds with an array of topic objects, each with slug and description properties', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.topics).toEqual(topicData);
```

Comment:
Hi Jodie, thanks for the feedback. Can you explain this a bit more please? What should I be comparing if I use one of those methods? Just comparing one topic object to one example topic object? And why does this make the testing more robust compared to comparing directly to the data? In this case where no serial key or other columns are added by the database, it seems like it would have the same result either way?